classifiers/nlp/ZeroShotTFClassifier/__init__.py

```diff
@@ -0,0 +1,236 @@
+import numpy as np
+
+from typing import List, Optional
+
+from jina.executors.classifiers import BaseClassifier
+from jina.executors.decorators import batching, as_ndarray
+from jina.executors.devices import TFDevice
+
+
+class ZeroShotTFClassifier(TFDevice, BaseClassifier):
+    """
+    :class: `ZeroShotTFClassifier` wraps tensorflow transformers from
+        huggingface, and performs zero shot learning classification
+        for nlp data.
+    """
+
+    def __init__(
+        self,
+        labels: List[str],
+        pretrained_model_name_or_path: str = "distilbert-base-uncased",
+        base_tokenizer_model: Optional[str] = None,
+        pooling_strategy: str = "mean",
+        layer_index: int = -1,
+        max_length: Optional[int] = None,
+        *args,
+        **kwargs,
+    ):
+        """
+        :param labels: the potential labels for the classification
```

Comment:
Our docstring guidelines suggest to put the `__init__` params at the class level and not in the constructor.